instrumentation/micrometer/micrometer-1.5/library/README.md

```diff
@@ -0,0 +1,41 @@
+# Micrometer Instrumentation for Micrometer version 1.5 and higher
+
+This module provides a Micrometer [registry](https://logback.qos.ch/manual/appenders.html) which
+sends Micrometer metrics to the
+[OpenTelemetry Metrics SDK](https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk/metrics).
+
+## Quickstart
+
+### Add these dependencies to your project
+
+Replace `OPENTELEMETRY_VERSION` with the [latest
+release](https://search.maven.org/search?q=g:io.opentelemetry.instrumentation%20AND%20a:opentelemetry-micrometer-1.5).
+
+For Maven, add to your `pom.xml` dependencies:
+
+```xml
+
+<dependencies>
+  <dependency>
+    <groupId>io.opentelemetry.instrumentation</groupId>
+    <artifactId>opentelemetry-micrometer-1.5</artifactId>
+    <version>OPENTELEMETRY_VERSION</version>
+  </dependency>
+</dependencies>
+```
+
+For Gradle, add to your dependencies:
+
+```groovy
+implementation("io.opentelemetry.instrumentation:opentelemetry-micrometer-1.5:OPENTELEMETRY_VERSION")
+```
+
+### Usage
+
+The instrumentation library provides an implementation of `MeterRegistry` to bridge Micrometer API to OpenTelemetry Metrics.
+
+```java
+void addOpenTelemetryMeterRegistry(OpenTelemetry openTelemetry) {
+    MeterRegistry meterRegistry = OpenTelemetryMeterRegistry.builder(openTelemetry).build();
+    Metrics.addRegistry(meterRegistry);
+}
```

Comment:
do you need to add ``` to close the code block?
This would work only if you use the global registry, do we also need to mention that?

Reply:
> This would work only if you use the global registry, do we also need to mention that?

good point, I generalized it to just show how to create the MeterRegistry for now